scripts/test/workflow.sh

```diff
@@ -18,6 +18,12 @@ if [[ $(git diff --name-only HEAD..develop|awk -F. '{print $NF}'|uniq) == md ]];
   exit
 fi
 
+if ! docker pull docker/whalesay
+then
+    echo "docker pull docker/whaleasy failed"
+    exit 1
+fi
```

Comment:
Giving the `set -e` in L15, do we need this `if` structure? Could we simply write

```bash
docker pull docker/whalesay
```